services/core-web/src/components/mine/NoticeOfWork/MineCard.js

```diff
@@ -42,13 +42,21 @@ export const MineCard = (props) => {
               </div>
               <div className="inline-flex padding-small">
                 <p className="field-title">Permit Number</p>
-                <ul className="mine-list__permits">
-                  {props.mine.mine_permit_numbers && props.mine.mine_permit_numbers.length > 0
-                    ? props.mine.mine_permit_numbers.map((permit_no) => (
-                        <li key={permit_no}>{permit_no}</li>
-                      ))
-                    : Strings.EMPTY_FIELD}
-                </ul>
+                {props.mine?.mine_permit_numbers?.length > 0 ? (
```

Comment:
this works in javascript?

Reply:
Yep!... And we oughtta start using it more often! Meredith noticed it and said the same in a recent previous PR of mine